pinot-segment-local/src/main/java/org/apache/pinot/segment/local/realtime/converter/RealtimeSegmentConverter.java

```diff
@@ -129,7 +129,7 @@ public void build(@Nullable SegmentVersion segmentVersion, ServerMetrics serverM
       recordReader.init(_realtimeSegmentImpl, sortedDocIds);
       RealtimeSegmentSegmentCreationDataSource dataSource =
           new RealtimeSegmentSegmentCreationDataSource(_realtimeSegmentImpl, recordReader);
-      driver.init(genConfig, dataSource, RecordEnricherPipeline.getPassThroughPipeline(),
+      driver.init(genConfig, dataSource, RecordTransformerPipeline.getPassThroughPipeline(),
```

Comment:
We want to integrate the current `RecordEnricher` as one type of `RecordTransformer`. With that, we shouldn't need a separate pipeline other than `TransformPipeline`